Libplanet/Net/Swarm.cs

```diff
@@ -1625,6 +1637,16 @@ CancellationToken cancellationToken
             }
 
             _logger.Debug("Finished to execute actions.");
+            if (Log.IsEnabled(LogEventLevel.Debug))
+            {
+                TimeSpan spent = DateTimeOffset.Now - executionStarted;
+                _logger.Debug(
```

Comment:
These lines should be also at a verbose level? @longfin

Reply:
Yup :)